TriblerGUI/widgets/downloadspage.py

```diff
@@ -380,6 +381,31 @@ def on_explore_files(self):
                                                  selected_item.download_info["destination"]))
             QDesktopServices.openUrl(QUrl.fromLocalFile(path))
 
+    def on_move_files(self):
+        if len(self.selected_items) != 1:
+            return
+
+        dest_dir = QFileDialog.getExistingDirectory(self, "Please select the destination directory",
+                                                    self.selected_items[0].download_info["destination"],
+                                                    QFileDialog.ShowDirsOnly)
+
+        _infohash = self.selected_items[0].download_info["infohash"]
+        _name = self.selected_items[0].download_info["name"]
+
+        data = {
+            "state": "move_storage",
+            "dest_dir": dest_dir
+        }
+
+        request_mgr = TriblerRequestManager()
+        request_mgr.perform_request("downloads/%s" % _infohash,
+                                    lambda res, _, name=_name, target=dest_dir: self.on_files_moved(res, name, target),
+                                    data=data, method='PATCH')
+
+    def on_files_moved(self, response, name, dest_dir):
+        if "modified" in response and response["modified"]:
+            self.window().tray_show_message(name, "Moved to %s" % dest_dir)
```

Comment:
Could you make this message a bit more descriptive? Something like:

"Moved download big-buck-bunny-1080p.avi to /my/new/path"

(so adding the name of the download)

Reply:
The name of the torrent is already present as the title of the tray message